src/plugins/discover/public/application/main/utils/fetch_all.ts

```diff
@@ -90,6 +90,7 @@ export function fetchAll(
     () => sendCompleteMsg(dataSubjects.main$, true),
     (error) => {
       if (error instanceof Error && error.name === 'AbortError') return;
+      if (error instanceof Error && error.name === 'ChartLoadingError') return;
```

Comment:
This would prevent data plugin error toasts for failures on the chart query. Those toasts should still show, since this is the place e.g. the Shard Failure modal is rendered.

I've made some overall changes to the way we do the querying in Discover in https://github.com/elastic/kibana/pull/120585. If we'd merge that first, we should not need any adjustments in this PR to the way we do querying.

Reply:
@majagrubic no that #120585 is merged , you can remove fetching logic adaptations in this PR 🥳 
Note that it would make sense to add an error icon with Popover to the number of hits being displayed for data view without timeField in a follow up PR